doc/reference/configuration/configuration_reference.rst

```diff
@@ -2988,6 +2988,73 @@ The ``memtx`` section is used to configure parameters related to the :ref:`memtx
     | Default: box.NULL
     | Environment variable: TT_MEMTX_SORT_THREADS
 
+
+
+
+..  _configuration_reference_metrics:
+
+metrics
+-------
+
+The ``metrics`` section defines configuration parameters for :ref:`metrics <monitoring>`.
+
+..  NOTE::
+
+    ``metrics`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+-   :ref:`metrics.exclude <configuration_reference_metrics_exclude>`
+-   :ref:`metrics.include <configuration_reference_metrics_include>`
+-   :ref:`metrics.labels <configuration_reference_metrics_labels>`
+
+
+.. _configuration_reference_metrics_exclude:
+
+..  confval:: metrics.exclude
+
+    Metrics to turn off.
+    This option accepts the same values as the ``exclude`` configuration parameter passed to :ref:`metrics.cfg() <metrics_cfg>`.
```

Comment:
Well, actually it's a bit different. It is allowed to use `metrics.cfg{exclude = 'all'}`, but configuration allows only table values `metrics: {exclude: ['all']}`. Same for include.